shared/desktop/renderer/renderer.html

```diff
@@ -1,14 +1,18 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="utf-8" http-equiv="Content-Security-Policy" content="default-src file: filesystem: http://localhost:4000 https://keybase.io/; img-src data: http://localhost:4000 https://keybase.io/ https://pbs.twimg.com/ https://avatars.githubusercontent.com/ https://images.coinbase.com/ https://s3.amazonaws.com/keybase_processed_uploads/ file:; style-src 'unsafe-inline'; script-src file: http://localhost:4000 chrome-extension://react-developer-tools 'unsafe-eval'; connect-src ws://localhost:4000 http://localhost:4000" />
-    <title>Keybase</title>
-  </head>
-  <body>
-    <div id="root">
-      <div title="loading..." style="flex: 1;background-color: #f5f5f5"></div>
-    </div>
-    <div id="modal-root"></div>
-    <script src="renderer-load.js"></script>
-  </body>
-</html>
+
+<head>
+  <meta charset="utf-8" http-equiv="Content-Security-Policy" content="default-src file: filesystem: http://localhost:4000 https://keybase.io/; img-src data: http://localhost:4000 http://127.0.0.1:* https://keybase.io/ https://pbs.twimg.com/ https://avatars.githubusercontent.com/ https://images.coinbase.com/ https://s3.amazonaws.com/keybase_processed_uploads/ file:; style-src 'unsafe-inline'; script-src file: http://localhost:4000 chrome-extension://react-developer-tools 'unsafe-eval'; connect-src ws://localhost:4000 http://localhost:4000"
```

Comment:
local isn't https? does it matter?

Reply:
Don't think it matters, would need root to get any traffic there.